content/providers/quick-start/data.yaml

```diff
@@ -340,85 +340,112 @@
     url: https://aws.amazon.com/braket/
 - codeExamples:
   - fullCode: |-
+      import qiskit
+      from qiskit.circuit.library import QuantumVolume
+
       from qiskit_aqt_provider import AQTProvider
-      aqt = AQTProvider('MY_TOKEN')
-      backend = aqt.get_backend('aqt_innsbruck')
 
-      # Build circuit
-      from qiskit.circuit.library import QuantumVolume
+      # Define a circuit
       circuit = QuantumVolume(5)
+      circuit.measure_all()
 
-      # Transpile circuit
-      from qiskit import transpile
-      transpiled_circuit = transpile(circuit, backend)
-      transpiled_circuit.draw()
-    name: Transpile
+      # Select an execution backend
+      provider = AQTProvider("ACCESS_TOKEN")
+      backend = provider.get_backend("offline_simulator_no_noise")
+
+      # Transpile the circuit to target the selected backend
+      transpiled_circuit = qiskit.transpile(circuit, backend, optimization_level=2)
+      print(transpiled_circuit)
+
+      # Execute the circuit on the selected backend
+      result = backend.run(transpiled_circuit, shots=50).result()
+      print(result.get_counts())
+    name: Transpile and execute
     runMethod: backend
   - fullCode: |-
+      from qiskit import QuantumCircuit
+
       from qiskit_aqt_provider import AQTProvider
-      from qiskit.primitives import BackendSampler
-      aqt = AQTProvider('MY_TOKEN')
-      backend = aqt.get_backend('aqt_innsbruck')
-      sampler = BackendSampler(backend)
+      from qiskit_aqt_provider.primitives import AQTSampler
 
-      # Build circuit
-      from qiskit import QuantumCircuit
-      circuit = QuantumCircuit(2, 2)
+      # Define a circuit
+      circuit = QuantumCircuit(2)
       circuit.h(0)
-      circuit.cx(0,1)
-      circuit.measure([0,1], [0,1])
+      circuit.cnot(0, 1)
+      circuit.measure_all()
```

Comment:
Please see my comment above for the provider list code example for the same code.

Reply:
Done.